sections/getting_started.tex

```diff
@@ -2,31 +2,24 @@ \section{Getting Started}
 
 \subsection{Installing Rust}
 
-\begin{frame}[c]{Rustup.rs}
-\begin{quote}
-«rustup is an installer for \\
-the systems programming language Rust» \\
-\vspace{0.5em}
-{\normalfont \small --- www.rustup.rs}
-\end{quote}
-\end{frame}
+\begin{frame}[fragile]{Rustup}
+	The best way to install and update the Rust toolchain is rustup
+	(\url{https://rustup.rs/}).
+
+	It makes it easy to:
+
+	\begin{itemize}
+		\item Install different Rust versions
+		\item Install cross compilation toolchains
+	\end{itemize}
+
+	\vspace{1em}
+
+	To download the installer script:
 
-\begin{frame}[fragile]{Rustup.rs}
-\begin{itemize}
-    \item Official installation method by now
-    \item Makes it easy to:
-      \begin{itemize}
-        \item Install different Rust versions
-        \item Install cross compilation toolchains
-      \end{itemize}
-    \item Written in Rust itself
-    \pause\item Installing is easy:
 \begin{minted}{sh}
-$ curl https://sh.rustup.rs -sSf | sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs
```

Comment:
This will just echo it to stdout, no? Either pipe to `sh` or pipe it to a script.

Reply:
Well yes, but that should be obvious. This way the user can decide what to do (either pipe through sh or send it to a file).

If you prefer we could also add `-o rustup.sh` to write it to a file.

Reply:
Added -o rustup.sh